content/en/observability_pipelines/scaling_and_performance/best_practices_for_scaling_observability_pipelines.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Best Practices for Scaling Observability Pipelines
-
+aliases:
+    - /observability_pipelines/best_practices_for_scaling_observability_pipelines/
 ---
 
 {{< site-region region="gov" >}}
```

Comment:
You can remove this alert as it's duplicated

Reply:
Removed